src/wrapped/wrappedopenal.c

```diff
@@ -63,9 +63,7 @@ void my_alRequestFoldbackStop(x64emu_t* emu);
 
 #include "wrappedlib_init.h"
 
-
-
-void fillALProcWrapper()
+void fillALProcWrapper(box64context_t* context)
```

Comment:
Don't change the signature of this function. Use my_context if you nedd the context (but for what?)

Reply:
I changed it because of the compiler warning, so it compiled without the warning, I didn't know it was intentional to leave it as it was

Reply:
/home/pablo/box64/src/wrapped/wrappedopenal.c:116:39: warning: too many arguments in call to 'fillALProcWrapper'
        fillALProcWrapper(emu->context);
        ~~~~~~~~~~~~~~~~~             ^
/home/pablo/box64/src/wrapped/wrappedopenal.c:116:26: warning: passing arguments to 'fillALProcWrapper' without a prototype is deprecated in all versions of C and is not supported in C2x [-Wdeprecated-non-prototype]
        fillALProcWrapper(emu->context);
                         ^
2 warnings generated.

Reply:
Well, I'll just return to how it was then

Reply:
it should be fixed at the warning, removog the parameter in call, that's the mystake